source/_components/cover.somfy_mylink.markdown

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: "Somfy MyLink Cover"
+description: "Instructions on how to integrate Somfy MyLink covers into Home Assistant."
+date: 2018-10-20 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tahoma.png
+ha_category: Cover
+ha_release: 0.XX
```

Comment:
Please use the next major version here (no placeholders).

Reply:
Have updated all files with v0.81